service/src/main/resources/db/migration/V355__application_document_refactoring.sql

```diff
@@ -0,0 +1,13 @@
+DROP VIEW IF EXISTS application_view;
+
+ALTER TABLE application
+    ADD COLUMN document     jsonb default '{}'                     NOT NULL,
+    ADD COLUMN form_modified timestamp with time zone default now() NOT NULL;
+
+UPDATE application a SET
+    document = (SELECT document FROM application_form af WHERE a.id = af.application_id AND af.latest = true),
+    form_modified = (SELECT created FROM application_form af WHERE a.id = af.application_id AND af.latest = true);
+
+DROP TABLE application_form;
```

Comment:
It might be good to keep this table (temporarily) to make it easier to revert the change if something breaks.